Sources/Tokenizer/Layout/Constraint.swift

```diff
@@ -16,8 +15,8 @@ public struct Constraint {
 }
 
 public enum ConstraintType {
-    case constant(Float)
-    case targeted(target: ConstraintTarget, targetAnchor: LayoutAnchor, multiplier: Float, constant: Float)
+    case constant(Double)
```

Comment:
Shouldn't be `Double` because the real implementation in AutoLayout uses `Float`. Applies to other places as well.